public/locales/en/translation.json

```diff
@@ -228,7 +241,8 @@
     "selectTool": "Select",
     "textTool": "Text",
     "title": "Tools",
-    "triangleTool": "Triangle"
+    "triangleTool": "Triangle",
+    "uploadTool": "Upload image"
```

Comment:
how about `imageTool`? I can see us adding support for other media / content types in the future, like video and audio.

Reply:
> I can see us adding support for other media / content types in the future, like video and audio.

Because of that I thought that „upload tool“ would be the name to go. Usually there are not different buttons per file type.

Reply:
this is obviously not a blocker - when we get to the point we support other media, this can be revisited.

Reply:
…I will do `s/uploadTool/imageTool/g`